src/frontend/src/features/rooms/components/Conference.tsx

```diff
@@ -47,6 +65,12 @@ export const Conference = ({
 
   const [showInviteDialog, setShowInviteDialog] = useState(mode === 'create')
 
+  useEffect(() => {
+    if (isFetchError) {
```

Comment:
do we want to create the room for any error or should check for more error details here?

Reply:
Already answer in commits or comments, we should improve this part to only triggers when the error is a 404.

Reply:
fixup